production-vms/main.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+
+	"production-vms/config"
+)
+
+// to run locally
+// 		production-vms % go run . --json-dir test-jsons
+
+func main() {
+	ctx, cancel := context.WithTimeout(context.Background(), config.Config.TestTimeout)
+	defer cancel()
+
+	jsonDir := flag.String("json-dir", "", "Directory containing JSON files")
```

Comment:
nit: consider using a dedicated struct for program opts that you combine with `flag.StringVar`, etc.